README.md

```diff
@@ -0,0 +1,734 @@
+
+# IBM Cloud Eventstreams Java SDK Version 1.1.0
+
+## Introduction
+
+IBM Event Streams for IBM Cloud™ is a high-throughput message bus built with Apache Kafka. 
+It is optimized for event ingestion into IBM Cloud and event stream distribution between your services and applications.
+
+Event Streams provides a REST API to help connect your existing systems to your Event Streams Kafka cluster. 
+Using the API, you can integrate Event Streams with any system that supports RESTful APIs.
+
+Documentation [IBM Cloud Eventstreams Service APIs](https://cloud.ibm.com/apidocs/event-streams).
+
+This is the Eventstreams Software Development Kit for `Java`
+It includes a library of functions used to access a Eventstreams cluster.
+
+## Table of Contents
+
+<!--
+  The TOC below is generated using the `markdown-toc` node package.
+
+      https://github.com/jonschlinkert/markdown-toc
+
+  You should regenerate the TOC after making changes to this file.
+
+      npx markdown-toc --maxdepth 4 -i README.md
+  -->
+
+<!-- toc -->
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Using the SDK](#using-the-sdk)
+- [REST API documentation](#event-streams-administration-rest-api)
+- [Questions](#questions)
+- [Issues](#issues)
+- [Open source @ IBM](#open-source--ibm)
+- [Contributing](#contributing)
+- [License](#license)
+
+<!-- tocstop -->
+
+## Overview
+
+The IBM Cloud Eventstreams SDK Java SDK allows developers to programmatically interact with the following IBM Cloud services:
+
+Service Name | Artifact Coordinates
+--- | ---
+[Admin Rest](https://cloud.ibm.com/apidocs/event-streams) | AdminRest
+
+## Prerequisites
+
+* An [IBM Cloud](https://cloud.ibm.com/registration) account.
+* The [IBM Cloud CLI.](https://cloud.ibm.com/docs/cli?topic=cli-getting-started)
+* An IAM API key to allow the SDK to access your account. Create one [here](https://cloud.ibm.com/iam/apikeys).
+* A IBM Cloud Eventstreams Instance Create one [here](https://cloud.ibm.com/registration?target=/catalog/services/event-streams)
+* Java 8 or above.
+
+## Installation
+The current version of this SDK is: 1.1.0
+
+Each service's artifact coordinates are listed in the table above.
+
+The project artifacts are published on the public [Maven Central](https://repo1.maven.org/maven2/)
+artifact repository.  This is the default public repository used by maven when searching for dependencies.
+To use this repository within a gradle build, please see
+[this link](https://docs.gradle.org/current/userguide/declaring_repositories.html).
+
+To use a particular service, define a dependency that contains the
+artifact coordinates (group id, artifact id and version) for the service, like this:
+
+##### Maven
+
+```xml
+<dependency>
+    <groupId>com.ibm.cloud</groupId>
+    <artifactId>eventstreams_sdk</artifactId>
+1.1.0
```

Comment:
Surely that's not valid xml!

Reply:
fixed in https://github.ibm.com/mhub/admin-rest-sdk/pull/50